client/src/pages/Messages/Messages.tsx

```diff
@@ -0,0 +1,44 @@
+import { Grid } from '@material-ui/core';
+import { useEffect, useState } from 'react';
+import NavBar from '../../components/NavBar/NavBar';
+import { useAuth } from '../../context/useAuthContext';
+import { useConversation } from '../../context/useConversationContext';
+import ChatBox from './ChatBox';
+import { ChatSideBar } from './ChatSideBar';
+import useStyles from './useStyles';
+
+const Messages = (): JSX.Element => {
+  const [profileId, setProfileId] = useState<string | null>(null);
+
+  const { conversations } = useConversation();
+
+  const { root } = useStyles();
+  const { loggedInUser } = useAuth();
+
+  const saveProfileId = (id: string) => {
+    setProfileId(id);
+  };
+
+  useEffect(() => {
+    let active = true;
+    loggedInUser && loggedInUser.profile && saveProfileId(loggedInUser.profile);
```

Comment:
Use optional chaining to shorten line: `loggedInUser?.profile && saveProfileId(...`